util/network-devp2p/Cargo.toml

```diff
@@ -10,12 +10,14 @@ edition = "2018"
 [dependencies]
 ansi_term = "0.11"
 bytes = "0.4"
+enr = "0.1.0-alpha-1"
 ethcore-io = { path = "../io", features = ["mio"] }
 ethereum-types = "0.8.0"
 igd = "0.10.0"
 ipnetwork = "0.12.6"
 keccak-hash = "0.4.0"
 libc = "0.2.7"
+libsecp256k1 = "0.3"
```

Comment:
Why use this lib rather than [`secp256k1`](https://github.com/rust-bitcoin/rust-secp256k1) which is what `parity-crypto` uses?

Reply:
This is what `enr` crate uses so the conversion between the two is necessary.